src/lib/components/HoverCard/root.svelte

```diff
@@ -0,0 +1,54 @@
+<script lang='ts' context="module">
+    export type HoverCardRootProps = BaseProps & {
+        /** The controlled open state of the hover card.*/
+        open?: boolean;
+        /** The duration (in ms) from when the mouse enters the trigger until the hover card opens. */
+        openDelay?: number;
+        /** The duration (in ms) from when the mouse leaves the trigger or content until the hover card closes. */
+        closeDelay?: number;
+    };
+
+    export type Context = {
+        open: boolean;
+
+        readonly openDelay: number;
+        readonly closeDelay: number;
+    };
+
+    const {getContext, setContext} = reactiveContext<Context>();
+    export const getRootContext = getContext;
+</script>
+
+<script lang='ts'>
+	import type { BaseProps } from "$lib/internal/types";
+
+    import {reactiveContext} from '$lib/internal/helpers/reactiveContext';
+    import {useActions} from '$lib/internal/helpers/useActions';
+
+    import {Popper} from '$lib/internal/components';
+
+    type $$Props = HoverCardRootProps;
+
+    export let open = false;
+    export let openDelay = 750;
+    export let closeDelay = 300;
+    export let use: $$Props['use'] = [];
+
+    const contextStore = setContext({
+        open: [false, v => open = v],
+        openDelay: [openDelay],
+        closeDelay: [closeDelay]
+    });
+
+    $: contextStore.set({
+        open,
+        openDelay,
+        closeDelay
+    });
+</script>
+
+<Popper.Root>
+    <div use:useActions={use ?? []} {...$$restProps} data-state={$contextStore.open ? 'open' : 'close'}>
```

Comment:
Why not pass it the props to Popper.Root? If it's missing a prop (e.g. use), we can add them